src/App.tsx

```diff
@@ -1,24 +1,18 @@
-import { Store } from './types/index';
-import {
-  RouterProvider,
-  BrowserRouter,
-  Route,
-  createRoutesFromElements,
-  createBrowserRouter
-} from 'react-router-dom'
+import { RouterProvider } from 'react-router-dom';
 import store from './store';
 import { Provider } from 'react-redux';
-import Layout from './future-hopr-lib-components/Layout';
-
-import router, { applicationMap } from './router';
+import router from './router';
 import WagmiProvider from './providers/Wagmi';
 import Watcher from './providers/Watcher';
+import { ToastContainer } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
 
 function App() {
   return (
     <Provider store={store}>
       <WagmiProvider>
         <Watcher />
+        <ToastContainer position="bottom-right" />
```

Comment:
Maybe adding this to our Layout can be better? just guessing

>Remember to render the ToastContainer once in your application tree. If you can't figure out where to put it, rendering it in the application root would be the best bet.

Reply:
In this case I think it is our decision. App should only be rendered once in the app so it would be fine if that is the requirement

Reply:
Looks good for me here